chart/values.yaml

```diff
@@ -255,12 +255,12 @@ postgresql:
 
 # https://github.com/bitnami/charts/tree/master/bitnami/redis#parameters
 redis:
-  # -- you must set a password; the password generated by the redis chart will be
-  # rotated on each upgrade:
-  password: ""
-  # you can also specify the name of an existing Secret
-  # with a key of redis-password set to the password you want
-  # auth:
+  auth:
+    # you must set a password; the password generated by the redis chart will be
```

Comment:
For the sake of [helm-docs](https://github.com/norwoodj/helm-docs#valuesyaml-metadata) it'd be nice to retain the `--` so they continue to render like before

Reply:
Makes sense, should be fixed now!